src/components/PriceLine.tsx

```diff
@@ -54,32 +58,72 @@ const PriceLine = ({
 
     if (!showBarrier) return null;
 
-    const width = showBarrierDragLine ? priceLineWidth + 16 : priceLineWidth;
+    const width = priceLineWidth + 12;
 
     return (
-        <div className='barrier-area' style={{ top: 0 }} ref={setDragLine} hidden={!visible}>
+        <div
+            className={classNames('barrier-area', { 'barrier-area--zero': isOverlappingWithPriceLine })}
+            style={{ top: 0 }}
```

Comment:
if it has `className`, will it make sense to move `style={{ top: 0 }} `inside of css?